code/__DEFINES/client.dm

```diff
@@ -0,0 +1,11 @@
+#define AUTH_CLIENT_VERB(verb_name, args...)\
+/client/collect_client_verbs(){\
+	. = ..();\
```

Comment:
This probably won't be an issue, but it's worth noting that there is a limit of how many times you can do this before it breaks. I believe the limit is something like 255